tests/fault_tolerance/etcd_ha/test_vllm.py

```
@@ -443,3 +448,147 @@ def test_etcd_non_ha_shutdown_vllm_disaggregated(
+        try:
+            requests.post(
+                f"http://localhost:{FRONTEND_PORT}/v1/completions",
+                json={
+                    "model": FAULT_TOLERANCE_MODEL_NAME,
+                    "prompt": "Tell me a very long story.",
+                    "max_tokens": ZOMBIE_INFLIGHT_MAX_TOKENS,
+                    "temperature": 0.0,
+                },
+                timeout=600,
+            )
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)